packages/dart/npt_flutter/lib/features/onboarding/util/profile_progress_listener.dart

```diff
@@ -4,18 +4,74 @@ import 'dart:developer';
 import 'package:at_client_mobile/at_client_mobile.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:npt_flutter/app.dart';
+import 'package:npt_flutter/constants.dart';
 import 'package:npt_flutter/features/profile_list/bloc/profile_list_bloc.dart';
+import 'package:npt_flutter/util/uuid.dart';
 
 import '../../profile_list/cubit/sync_cubit.dart';
 
 class ProfileProgressListener extends SyncProgressListener {
```

Comment:
The spike approach was to subscribe to notifications auto-generated as a result of another atSign sharing or unsharing a profile. Receiving an `update` triggers a `sync` the results of which are later handled by the `onSyncProgressEvent` function below; receiving a `delete` removes the `cached:` profile and triggers a `ProfileListLoadEvent` for the `ProfileListBlock`